Makefile

```diff
@@ -195,19 +195,37 @@ docker-buildx-dev: test
 	@if ! DOCKER_CLI_EXPERIMENTAL=enabled docker buildx ls | grep -q container-builder; then\
 		DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name container-builder --use;\
 	fi
-	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
-		-t $(REPOSITORY):$(DEV_TAG) \
-		-t $(REPOSITORY):dev \
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/amd64" \
+		-t $(REPOSITORY):$(DEV_TAG)-linux-amd64 \
+		. --push
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/arm64" \
+		-t $(REPOSITORY):$(DEV_TAG)-linux-arm64 \
+		. --push
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/arm/v7" \
+		-t $(REPOSITORY):$(DEV_TAG)-linux-armv7 \
 		. --push
+	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create $(REPOSITORY):$(DEV_TAG) $(REPOSITORY):$(DEV_TAG)-linux-amd64 $(REPOSITORY):$(DEV_TAG)-linux-arm64 $(REPOSITORY):$(DEV_TAG)-linux-armv7
+	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create $(REPOSITORY):dev $(REPOSITORY):$(DEV_TAG)-linux-amd64 $(REPOSITORY):$(DEV_TAG)-linux-arm64 $(REPOSITORY):$(DEV_TAG)-linux-armv7
+	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push $(REPOSITORY):$(DEV_TAG)
+	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push $(REPOSITORY):dev
```

Comment:
I didn't see this in my previous PR until now, we don't need both these to be pushed

```
	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push $(REPOSITORY):$(DEV_TAG)
	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push $(REPOSITORY):dev
```

Makefile L7 `DEV_TAG ?= dev` which means you're effectively just pushing the same tag twice.

Reply:
It's not pushing the same tag twice. If you look at the comment above L7, DEV_TAG gets overriden by 7-digit SHA. It's pushing SHA tag + `dev` tag (so it's easier to grab the latest dev tag)